src/content/learning/using-durable-objects.md

```diff
@@ -0,0 +1,216 @@
+---
+order: 8
+---
+
+# Using Durable Objects
+
+Durable Objects provide low-latency coordination and consistent storage for the Workers platform through two features: global uniqueness and a transactional storage API.
+
+* Global Uniqueness guarantees that there will be a single Durable Object with a given id running at once.  Requests for a Durable Object id are routed by the Workers runtime to the Cloudflare point-of-presence that owns the Durable Object.  
+
+* The transactional storage API provides strongly-consistent, key-value storage to the Durable Object.  Each Object can only read and modify keys associated with that Object. Execution of a Durable Object is single-threaded, but multiple request threads may be processed out-of-order from how they arrived at the Object.
+
+## Using Durable Objects
+
+Durable Objects are named instances of a class you define.  Just like a class in object-oriented programming, the class defines the methods and data a Durable Object can access.
+
+There are four steps to creating a Durable Object:
+
+* __Writing the class__ that defines a Durable Object.
+* __Binding that class__ into a Worker.
+* __Instantiating a Durable Object__ from within a running Worker.
+* __Communicating with a Durable Object__ from a running Worker via the Fetch API.
+
+## Writing a class that defines a Durable Object
+
+To define a Durable Object, developers export an ordinary Javascript class.  Other languages will need a shim that translates their class definition to a Javascript class.
+
+The first variable passed to the class constructor contains state specific to the Durable Object, including methods for accessing storage.
```

Comment:
The natural next question that doesn't appear to be answered here is what's the second variable passed to the constructor?